templates/focus-ios.xliff

```diff
@@ -201,7 +201,6 @@
       </trans-unit>
       <trans-unit id="BiometricPrompt.reason">
         <source>Unlock %@ when re-opening in order to prevent unauthorized access.</source>
-        <note>%@ is app name. Explanation for why the app needs access to biometric information. Prompt is only shown once when the user first tries to enable Face ID to open the app.</note>
```

Comment:
Sorry misclicked and approved the PR.
COuld you explain why you're removing a localization comment please?